HomeTask1_1/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace HomeTask1_1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            const string Digits = "0123456789AB";
+
+            Console.WriteLine("Enter first number - ");
+            string firstNumberInput = Console.ReadLine();
+
+            Console.WriteLine("Enter second number - ");
+            string secondNumberInput = Console.ReadLine();
+
+            int firstNumber = int.Parse(firstNumberInput);
+            int secondNumber = int.Parse(secondNumberInput);
+
+            for (int i = firstNumber; i <= secondNumber; i++)
+            {
+                string calculateValue = "";
+                int currentValue = i;
+                int count = 0;
+
+                while (currentValue > 0)
+                {
```

Comment:
If you do not implement the separate method for converting to duodecimal system you can calculate number of "A" directly here, not converting the whole number (according to the task it s not required)